gdx/src/com/badlogic/gdx/graphics/Mesh.java

```diff
@@ -1027,7 +1027,7 @@ public Mesh copy (boolean isStatic, boolean removeDuplicates, final int[] usage)
 					if (a == null) continue;
 					for (int j = 0; j < a.numComponents; j++)
 						checks[++idx] = (short)(a.offset + j);
-					attrs[++ai] = new VertexAttribute(a.usage, a.numComponents, a.alias);
+					attrs[++ai] = new VertexAttribute(a.usage, a.numComponents, a.type, a.normalized, a.alias, a.unit);

```

Comment:
Looks like this would be better suited in a copy method. That would also hold when the class is extended.